packages/lwc-language-server/src/lwc-data-provider.ts

```diff
@@ -45,8 +45,8 @@ export class LWCDataProvider implements IHTMLDataProvider {
     }
     provideValues(): IValueData[] {
         const values: IValueData[] = [];
-        this.indexer.customData.forEach(t => {
-            t.classMembers.forEach(cm => {
+        this.indexer.customData?.forEach(t => {
+            t.classMembers?.forEach(cm => {
```

Comment:
can we add a unit test for this condition? One to cover where customData isn't defined and one for when t.classMembers isn't defined?

Reply:
I knew you were gonna ask for it 😆

Reply:
I added a unit test for the case when t.classMembers is undefined.

For indexer.customData, indexer.customData() is a wrapper around `ComponentIndexer.tags`, and tags can never be null (in the body of the class, it's defined as `readonly tags: Map<string, Tag> = new Map();`).